winterfell/webapp/.eslintignore

```diff
@@ -0,0 +1,2 @@
+src/libs/**
```

Comment:
supposedly the top level eslintignore file is supposed to cascade down but in practice i wasnt seeing this happen, will dig in more